src/extension-client/remix-extension.ts

```diff
@@ -16,8 +20,20 @@ export class RemixExtension<T extends Api = any> {
   protected currentRequest: PluginRequest
   public isLoaded = false
 
-  constructor() {
+  constructor(opts: RemixExtensionOptions) {
     window.addEventListener('message', event => this.getMessage(event), false)
+
+    if (!opts.useCustomBootStrapTheme) this.initListenOnSwitchTheme()
+  }
+
+  private initListenOnSwitchTheme () {
+    let cssLink = document.createElement('link')
+    cssLink.setAttribute('rel', 'stylesheet')
+    document.head.appendChild(cssLink)
+    this.listen('theme', 'switchTheme', (theme) => {
```

Comment:
Type `theme` if you know the properties : `(theme: {url: string, quality: 'dark' | 'light'})`